### PR TITLE
build aarch64-linux-musl binary for releases

### PR DESCRIPTION
**Pull request by usamoi**
generate binaries: https://github.com/usamoi/sqllogictest-rs/releases/tag/v0.0.0

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -68,10 +71,20 @@ jobs:
...
+        deb [arch=amd64] http://azure.archive.ubuntu.com/ubuntu jammy-updates main restricted universe multiverse
+        deb [arch=amd64] http://azure.archive.ubuntu.com/ubuntu jammy-security main restricted universe multiverse
+        deb [arch=arm64] http://ports.ubuntu.com/ubuntu-ports jammy main restricted universe multiverse
+        deb [arch=arm64] http://ports.ubuntu.com/ubuntu-ports jammy-updates main restricted universe multiverse
+        deb [arch=arm64] http://ports.ubuntu.com/ubuntu-ports jammy-security main restricted universe multiverse
+        EOL
         sudo apt-get update
-        sudo apt-get install -y --no-install-recommends xz-utils liblz4-tool musl-tools
+        sudo apt-get install -y --no-install-recommends xz-utils liblz4-tool musl-tools musl-dev musl-dev:arm64
+        echo "RUSTFLAGS=-Clinker=rust-lld" >> $GITHUB_ENV
```
Comment (xxchan) [inline .github/workflows/release.yml:87]: IIRC `rust-lld` is the default linker? :eyes:

**Review by xxchan — APPROVED** [review]
Thanks!

**Comment by xxchan** [comment]
BTW, may I ask what are you using `sqllogictest-rs` for? :eyes: You may also add to "Used By" https://github.com/risinglightdb/sqllogictest-rs?tab=readme-ov-file#used-by if you don't mind.

File: .github/workflows/release.yml (hunk 1 above)
Comment (usamoi) [reply .github/workflows/release.yml:87]: I think `rust-lld` is default linker on `x86_64-linux-gnu`, not `aarch64-linux-musl`.

**Comment by xxchan** [comment]
Released https://github.com/risinglightdb/sqllogictest-rs/releases/tag/v0.25.0 @usamoi

**Pull request merged**